drivers/clock_control/clock_control_esp32.c

```diff
@@ -687,14 +686,13 @@ static int esp32_cpu_clock_configure(const struct esp32_cpu_clock_config *cpu_cf
 	esp_cpu_set_cycle_count((uint64_t)esp_cpu_get_cycle_count() * rtc_clk_cfg.cpu_freq_mhz /
 				old_config.freq_mhz);
 
+#if !defined(ESP_CONSOLE_UART_NONE)
```

Comment:
just a note to check the CONFIG_ESP_CONSOLE_USB_SERIAL_JTAG case (if necessary)
as the routine `esp_rom_uart_set_clock_baudrate` mentions UART I couldn't be sure about the USB clock case

Reply:
This needs to be handled in another PR.